src/styles/blocks/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: $primary-text-color;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 1.5;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 1.4;
+    color: $secondary-text-color;
+    width: 100%;
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__counter {
+    font-size: 10px;
+    line-height: 1.4;
+    font-weight: 400;
+  }
+
+  &__price {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-title {
+      font-size: 12px;
+      line-height: 1.5;
+      color: $secondary-text-color;
+    }
+
+    &-price {
+      font-size: 16px;
+      line-height: 1.12;
+      font-weight: 700;
+    }
+  }
+
+  &__buy-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    width: 100%;
+    height: 40px;
+    background: $blue-accent;
+    border-radius: 5px;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 1.14;
+    text-transform: uppercase;
+  }
+
+  &:hover .card__buy-button {
+    border: 1px solid $blue-accent;
+    border-radius: 5px;
+    background-color: #fff;
+    color: $blue-accent;
+  }
+}
+
+.card--spacing:not(:nth-last-child(-n+4)) {
+  margin-bottom: 46px;
+}
```

move it to BEM block card
```suggestion
  }
  
  &--spacing:not(:nth-last-child(-n+4)) {
    margin-bottom: 46px;
  }
}
```